pkg/compactor/shard_aware_deduplicate_filter.go

```diff
@@ -158,10 +160,10 @@ func (f *ShardAwareDeduplicateFilter) DuplicateIDs() []ulid.ULID {
 //
 // - D with sources 1, 2, 3, 4, 5
 //
-// Then B is a successor of A (A sources are subset of B sources, but not vice versa), and D is successor of both A, B and C.
+// Then B is a successor of A (A sources are subset of B sources, but not vice versa), and D is a successor of all A, B and C.
```

Comment:
Why is "both" wrong here?

Reply:
@aknuds1 Is both supposed to be used also when there are more than 2 elements? (Here we have A, B and C)